src/app/dashboard/dashboard/dashboard.component.ts

```diff
@@ -41,24 +43,81 @@ export class DashboardComponent implements OnInit, OnDestroy {
   dragulaSubscription = new Subscription();
   constructor(
     public teamService: TeamService,
-    private dragulaService: DragulaService,
+    private dragularService: DragulaService,
+    private allocationsService: AllocationsService,
+    private projectsService: ProjectsService,
+    private skillsService: SkillsService,
     private overlayService: OverlayService,
-    private studentsService: StudentsService
-  ) {
-    /* save model when modified by drag & drop operation */
-    this.dragulaSubscription.add(
-      dragulaService.dropModel('persons').subscribe(({ target, item }) => {
-        const newTeam = teamService.getTeamByName(target.id);
-        item.teamName = target.id;
-        // if person belongs to no Team (Person Pool) newTeam is undefined
-        newTeam?.add(item);
-        teamService.saveToLocalBrowserStorage();
-      })
+    private studentsService: StudentsService,
+    private studentPersonTransformerService: StudentToPersonService,
+    private projectTeamTransformerService: ProjectToTeamService
+  ) {}
+
+  private _students: Student[];
+  private _projects: Project[];
+  private _skills: Skill[];
+  private _allocations: Allocation[];
+
+  studentsWithoutTeam: Student[];
+  dataLoaded = false;
+
+  ngOnInit(): void {
+    this.studentsService.students$.subscribe(students => {
```

Comment:
please unsubscribe. 
There are some different ways. I like to simply unsubscribe onDestroy: 
`studentSubscription: Subscription; `

`this.studentSubscription =  this.studentsService.students$.subscribe(students => { ... } `

```
ngOnDestroy() {
this.studentSubscription?.unsubscribe(); 
}
```

Reply:
Does this make sense, when our component is never "destroyed"?